core/src/main/scala/sttp/openai/requests/audio/transcriptions/TranscriptionConfig.scala

```diff
@@ -0,0 +1,35 @@
+package sttp.openai.requests.audio.transcriptions
+
+import sttp.openai.requests.audio.Model
+import sttp.openai.requests.images.ResponseFormat
+
+import java.io.File
+import java.nio.file.Paths
+
+case class TranscriptionConfig(
+    file: File,
+    model: Model,
+    prompt: Option[String] = None,
+    responseFormat: Option[ResponseFormat] = None,
+    temperature: Option[Float] = None,
+    language: Option[String] = None
+)
+
+object TranscriptionConfig {
+  def createTranscriptionConfigWithSystemPaths(
+      systemPathImage: String,
+      model: Model,
+      prompt: Option[String] = None,
+      responseFormat: Option[ResponseFormat] = None,
+      temperature: Option[Float] = None,
+      language: Option[String] = None
```

Comment:
**suggestion:** Please add a scaladoc describing how to format this string, with an example. This applies to `TranslationConfig` as well.